Doc/library/unittest.rst

```diff
@@ -1446,17 +1449,19 @@ along with their deprecated aliases:
     :meth:`.assertAlmostEqual`      failUnlessAlmostEqual  assertAlmostEquals
     :meth:`.assertNotAlmostEqual`   failIfAlmostEqual      assertNotAlmostEquals
     :meth:`.assertRegex`                                   assertRegexpMatches
+    :meth:`.assertNotRegex`                                assertNotRegexpMatches
```

Comment:
The `assertNotRegexpMatches` is not available for Python < 3.5, and we are saying that it has been deprecated in Python 3.2, when it was not available.  IMHO we should remove this reference.